### PR TITLE
[WIP] Update documentation to reference logging ansible work

### PR DESCRIPTION
**Pull request by jcantrill**
Do not merge.  This is in support of https://trello.com/c/zjOqOXET/324 and requires additional input changes based on 

1. missing equivalent features for component node selection 
2. troubleshooting recommendations

cc @bmcelvee @ewolinetz @richm

### CODE REVIEW

File: install_config/aggregate_logging.adoc [hunk 1]
```diff
@@ -95,147 +77,274 @@ deployed. To control component placement, specify node selectors per component t
...
-least specify the host name at which Kibana should be exposed to client
-browsers, and also the master URL where client browsers will be directed to
-for authenticating to {product-title}.
+| `*openshift_logging_use_ops*`
+| If set to `true`, configures a second Elasticsearch cluster and Kibana for
+operations logs. Fluentd splits logs between the main cluster and a cluster reserved
+for operations logs (which consists of *_/var/log/messages_* on nodes and the logs from the
+projects *default*, *openshift*, and *openshift-infra*).  This means a second Elasticsearch
+and Kibana are deployed. The deployments are distinguishable by the *-ops* suffix included
+in their names and have parallel deployment options listed below.
```
Comment (richm) [inline install_config/aggregate_logging.adoc]: The current ansible parameter for this is `openshift_hosted_logging_enable_ops_cluster`

**Comment by richm** [comment]
Does this need to explain which parameters are required vs. optional?  Should this explain what the default value is in the case of optional parameters?

File: install_config/aggregate_logging.adoc [hunk 2]
```diff
@@ -437,11 +478,9 @@ own storage, but an {product-title} deployment configuration shares storage
 volumes between all its pods. So, when scaled up, the EFK deployer ensures each
 replica of Elasticsearch has its own deployment configuration.
 
-It is possible to scale your cluster up after creation by adding more
-deployments from a template; however, scaling up (or down) requires
-the correct procedure and an awareness of clustering parameters (to be
-described in a separate section). It is best to indicate the
-desired scale at first deployment.
+It is possible to scale your cluster up after creation by modifying the
+`openshift_logging_es_cluser_size` in the inventory file and re-running the logging
```
Comment (richm) [inline install_config/aggregate_logging.adoc]: openshift_logging_es_cluster_size

File: install_config/aggregate_logging.adoc [hunk 3]
```diff
@@ -701,66 +652,51 @@ $ oc edit configmap/logging-elasticsearch
...
+// journal path. For example, if you want Fluentd to always read logs
+// from the transient in-memory journal, set `journal-source`=*_/run/log/journal_*.
 
-| `journal-read-from-head`
-|If this setting is *false*, Fluentd starts reading from the end of the journal,
-ignoring historical logs. If this setting is *true*, Fluentd starts reading
+| `openshift_logging_journal_read_from_head`
+|If this setting is `false`, Fluentd starts reading from the end of the journal,
+ignoring historical logs. If this setting is `true`, Fluentd starts reading
 logs from the beginning of the journal.
```
Comment (richm) [inline install_config/aggregate_logging.adoc]: The default value is `false`

File: install_config/aggregate_logging.adoc [hunk 4]
```diff
@@ -1000,8 +935,15 @@ $ oc scale dc/logging-kibana --replicas=2
...
-`KIBANA_HOSTNAME` variable.
+[NOTE]
+====
+To ensure the scale persists across multiple executions of the logging playbook,
+make sure to update the `openshift_logging_kibana_replica_count` in the inventory file.
+====
+
+
+You can see the user interface by visiting the site specified by the
+`openshift_logging_kibana_hostnaem` variable.
```
Comment (richm) [inline install_config/aggregate_logging.adoc]: openshift_logging_kibana_hostname

File: install_config/aggregate_logging.adoc (hunk 1 above)
Comment (jcantrill) [inline install_config/aggregate_logging.adoc]: The role 'openshift_hosted_logging' will intentionally go away in 3.5 in favor of 'openshift_logging' per the direction from @detiber when we took upon the new work.

File: install_config/aggregate_logging.adoc (hunk 2 above)
Comment (jcantrill) [inline install_config/aggregate_logging.adoc]: Updated with next commit

**Comment by jcantrill** [comment]
@richm We probably need to have a larger conversation about providing defaults or not.  I have a proof of concept to parse the defaults/main.yaml so we don't have to maintain the list which is a pain.  I would much prefer to generate this table to keep the values in sync.

File: install_config/aggregate_logging.adoc (hunk 3 above)
Comment (jcantrill) [inline install_config/aggregate_logging.adoc]: Updated

File: install_config/aggregate_logging.adoc [hunk 5]
```diff
@@ -1139,20 +1081,13 @@ $ oc rollout latest dc/logging-curator-ops
...
 
-[[aggregate-logging-upgrading]]
-== Upgrading
-
-To upgrade the EFK logging stack, see
-xref:../install_config/upgrading/manual_upgrades.adoc#manual-upgrading-efk-logging-stack[Manual
-Upgrades].
-
 [[troubleshooting-kibana]]
 == Troubleshooting Kibana
```
Comment (jcantrill) [inline install_config/aggregate_logging.adoc]: @richm @ewolinetz   Not reflected specifically at this line, but this section regarding troubleshooting basically says 'modify THIS value and rerun the deployer.'  What should be our message going forward with troubleshooting and ansible be.  Thoughts?  'Check for XXXXX, update inventory, rerun playbook...'

**Comment by jcantrill** [comment]
@bmcelvee We may wish to change this PR such that we fully display the old version for 'enterprise' and the 'new version' for origin since the deployment method is significant enought

**Comment by bmcelvee** [comment]
@jcantrill Thanks for the heads up. I'll look into the best way to make it happen for this PR as well.

**Comment by bmcelvee** [comment]
Continuing edits in https://github.com/openshift/openshift-docs/pull/3918.

**Pull request closed without merging**